client/client.js

```diff
@@ -85,15 +97,57 @@ Client.prototype.dataStore = function (message) {
   }
 
   if (changed) {
-    this.lastModified = Date.now();
-    Core.Persistence.persistKey(this.key, this, Client.dataTTL);
+    this._persist();
   }
 
   return true;
 };
 
+Client.prototype.dataLoad = function (callback) {
+  var self = this;
+
+  // When we touch a client, refresh the TTL
+  Core.Persistence.expire(self.key, Client.dataTTLGet());
+
+  // Update persisted subscriptions/presences
+  Core.Persistence.readKey(self.key, function (clientOld) {
+    if (clientOld) {
+      self.subscriptions = clientOld.subscriptions;
+      self.presences = clientOld.presences;
+
+      // Remove obsolete presences
+      Client._presencesDelete(self.presences, clientOld.id);
```

Comment:
I used this to remove the online/offline messages for socketIds that I know are already removed from the system, and yes, the system can take care of them as you suggest, so I'm fine with that.

Also, I removed the explicit buffering of messages in the earlier commit (9c17f877), and instead use the *pauseable* module to pause the inbound socket while old messages are being restored; the socket is resumed when the restore is done.  (This does the same buffering in the background that I was doing, but it is centralized in this single module.)

I have tested this code against the current 0.13.1 radar_client, and against the upcoming 0.14.0 radar_client, and for both clients the presences and subscriptions are the same in redis.